chapters/annotations.tex

```diff
@@ -1385,10 +1387,11 @@ \section{Annotations for the Graphical User Interface}\label{annotations-for-the
 The \lstinline!showStartAttribute = true! is primarily intended for non-parameter values and avoids introducing a separate parameter for the start-value of the variable.
 \end{nonnormative}
 
-If \lstinline!colorSelector = true!, it indicates that an rgb-value selector can be presented for a vector of three elements and generate values 0..255 (the color selector should be useable both for vectors of \lstinline!Integer! and \lstinline!Real!).
+If \lstinline!colorSelector = true!, it suggests that an rgb-value selector can be presented for a vector of three elements and generate values 0..255 (the color selector should be useable both for vectors of \lstinline!Integer! and \lstinline!Real!).
 
-A \fmtannotationindex{Selector} displays a file dialog to select a file.
-Setting \lstinline!filter! only shows files that fulfill the given pattern defined by \lstinline!text1 (*.ext1);;text2 (*.ext2)! to show only files with file extension \filename{ext1} or \filename{ext2} and displaying a description text \lstinline!text1! and \lstinline!text2!, respectively.
+The presence of \lstinline!loadSelector! or \lstinline!saveSelector! specifying \fmtannotationindex{Selector} suggests a file dialog to select a file.
```

Comment:
Looks like some words are missing (at least for the kind of suggestion that I find consistent with the rgb-value selector phrase above):
```suggestion
The presence of \lstinline!loadSelector! or \lstinline!saveSelector! specifying \fmtannotationindex{Selector} suggests use of a file dialog to select a file.
```

Variants that would also fill the gap: _suggests to use a_, _suggests using a_

Reply:
I think it should 'the use of' and I'll add that.

Reply:
OK.  Resolving.